spec/requests/rack_attack_spec.rb

```diff
@@ -160,7 +168,7 @@ def till_throttled(requests_to_send:, throttled_status: 429, mode: :default, &bl
       till_throttled(requests_to_send: throttled_at, throttled_status: -999) { get(path, headers: headers) }
       expect(SlackSendMonitor.lifetime_attempts).to be > 20
       expect(SlackSendMonitor.lifetime_sends).to be > 1
-      expect(SlackSendMonitor.percent_sent).to be < 10
+      expect(SlackSendMonitor.percent_sent).to be < 15
```

Comment:
was this test failing with ruby 3.3? That seems unexpected

Reply:
not specifically for ruby 3.3, but inconsistently on and off over the past few months.  I'll take another look. It may only need to be bumped to 11.  Sometimes it returns 10.3 or 10.4